src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package service.impl;
+
+import database.Storage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String WORDS_SEPARATOR = ",";
+    private static final String NEW_LINE = "\n";
+    private static final String HEAD_OF_FILE = "fruit,quantity\n";
+
+    @Override
+    public List<String> getReportList() {

```

Comment:
```suggestion
    public String getReportList() {
```
better return simple String